include/envs.h

```diff
@@ -50,6 +50,7 @@ static char *afl_environment_variables[] = {
     "AFL_FAST_CAL",
     "AFL_FORCE_UI",
     "AFL_FUZZER_ARGS",  // oss-fuzz
+    "AFL_GDB",
```

Comment:
What does it do?

Reply:
for aflpp_driver.c ... prints out the pid, waits a second and also dump the memory map from proc

Reply:
Should we add documentation for it?

Reply:
nah, it is rather niche, and I guess I am the only one using it ever